src/styles/TextFieldCatalog.scss

```diff
@@ -35,3 +35,14 @@
 .demo-text-field-outlined-shaped {
   @include mdc-text-field-outline-shape-radius(50%);
 }
+
+.hero-text-field-container {
+  display: flex;
+  flex: 1 1 auto;
+  justify-content: space-evenly;
```

Comment:
The children are not centered horizontally in Edge or IE, and not centered vertically in IE. `justify-content: space-around` should at least fix the horizontal alignment. (IE/Edge don't support `space-evenly`)